docs/rfcs/0002-storing-course-data.md

```diff
@@ -0,0 +1,134 @@
+## Storing course data
+
+### Abstract
+
+The aim of this RFC is to make decisions about the following:
+- The model definitions* that will be needed to store course data.
+- The plan for loading legacy course data, and keeping that data current
+  after we switch to using our own Open edX instance.
+
+Out of scope for this RFC:
+- Content management – we will just rely on Django admin for creating/updating
+  courses by hand.
+- Any kind of grading/certificate data.
+
+\* - _Much like MicroMasters, we will keep our own models for courses/programs
+  that will exist in parallel with the course data in edX_
+
+### Models
+
+Since this app has very similar needs, `Program`, `Course`, and `CourseRun` 
+models will very closely mirror the [models in MicroMasters](https://github.com/mitodl/micromasters/blob/master/courses/models.py), 
+minus a few fields that are not relevant for this app.
+
+`Program`
+- title
+- description
+- thumbnail (image)
```

Comment:
Currently in MicroMasters , is "thumbnail" field different from what gets displayed in MM front page ?

Reply:
Nope, that's the field for the image you would see in the front page. `thumbnail` is the name of the column in MM, perhaps it could be named more clearly